docs/core-packages/babylon.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: docs
+title: A JavaScript parser
+description: Babylon is a JavaScript parser used in Babel
+permalink: /docs/core-packages/babylon/
```

Comment:
I would consider it as one (especially since it used to be in the monorepo). Since Babel = parser, transformer(s), generator